modules/rosa-list-objects.adoc

```diff
@@ -480,6 +524,42 @@ Describe an add-on named `dbaas-operator`.
 ----
 $ rosa describe addon dbaas-operator
 ----
+[id="rosa-describe-break-glass-credential_{context}"]
+== describe break glass credential
+
+Shows the details for a break glass credential for a specific cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa describe break-glass-credential --break_glass_credential_id=<break_glass_credential_id> --cluster=<cluster_name>| <cluster_id> [arguments]
```

Comment:
The command must be "rosa describe break-glass-credential --id=<break_glass_credential_id> --cluster=<cluster_name>| <cluster_id> [arguments]"